src/20240727_AaveV3Arbitrum_GHOFlashMinterFacilitatorArbitrum/GHOFlashMinterFacilitatorArbitrum.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "GHO Flash Minter Facilitator Arbitrum"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-gho-flash-minter-facilitator-arbitrum/18445"
+snapshot: "TODO"
+---
+
+## Simple Summary
+
+This publication proposes deploying a FlashMinter Facilitator on Arbitrum.
+
+## Background
+
+1. **Title of Facilitator**
+   Arbitrum FlashMinter Facilitator
+
+2. **High-Level Description of Mechanism/Request**
+   FlashMinting provides the same functionality as a flashloan, but instead of borrowing assets from a pool, users would be able to FlashMint GHO and repay in a single transaction.
+
+3. **Author / Link to License**
```

Comment:
Please do preview of the md, as it does not reander the same as in the forum (from where it was taken)